features/polyExplorer/src/screens/stories/digitalGiantsStory.jsx

```diff
@@ -45,6 +46,99 @@ const DigitalGiantsStory = () => {
         ...new Set(jurisdictionLinks.map(({ target }) => target)),
     ].filter((j) => j !== "EU-GDPR");
 
+    const listOfDataCategories = Object.keys(
+        globalData.personal_data_categories
+    );
+
+    let totalShares = 0;
+    bigSix.forEach((company) => {
+        company._data.dataTypesShared.forEach((i) => (totalShares += i.count));
+    });
+
+    const dataTypesSharedCombined = listOfDataCategories
+        .map((category) => {
+            let total = 0;
+            bigSix.map((company) => {
+                company._data.dataTypesShared.forEach((typeCategory) => {
+                    if (typeCategory["dpv:Category"] === category)
+                        total += typeCategory.count;
+                });
+            });
+            return total !== 0
+                ? {
+                      "dpv:Category": category,
+                      total,
+                  }
+                : null;
+        })
+        .filter((e) => e)
+        .sort((a, b) => b.total - a.total);
+
+    const [selectedDataTypeBubble, setSelectedDataTypeBubble] = useState(
+        dataTypesSharedCombined[0].total
+    );
+
+    const dataTypes = [
+        {
+            id: "by-companies",
+            label: "By Companies",
+            translation: i18n.t(`${i18nHeader}:datatypes.tab.companies`),
+            bubbleColor: "#FB8A89",
+            bubbleTextColor: "#FB8A89",
+            bubbleStroke: "none",
+            data: bigSixNames.map((companyName, n) => {
+                return {
+                    title: companyName,
+                    bubbles: bigSix[n]._data.dataTypesShared.map(() => {
+                        return { value: 1 };
+                    }),
+                };
+            }),
+        },
+        {
+            id: "by-shares",
+            label: "By Shares",
+            translation: i18n.t(`${i18nHeader}:datatypes.tab.shares`),
+            bubbleColor: "#FB8A89",
+            bubbleTextColor: "#FB8A89",
+            bubbleStroke: "none",
+            data: bigSixNames.map((companyName, n) => {
+                return {
+                    title: companyName,
+                    bubbles: bigSix[n]._data.dataTypesShared.map((bubble) => {
+                        return { value: bubble.count };
+                    }),
+                };
+            }),
+        },
+        {
+            id: "by-types",
+            label: "By Types",
+            translation: i18n.t(`${i18nHeader}:datatypes.tab.types`),
+            bubbleColor: "#FB8A89",
```

Comment:
These literals are repeated throughout the text, have you considered defining them somewhere else?